clients/include/testing_hpr_strided_batched.hpp

```diff
@@ -90,7 +90,7 @@ void testing_hpr_strided_batched_bad_arg()
 template <typename T>
 void testing_hpr_strided_batched(const Arguments& arg)
 {
-    using U = decltype(std::real(T(0)));
+    using U = typename rocblas_real_type<T>::type;
```

Comment:
There is some way to write things (see my comments) where we should not have to specify typename here.

template < typename T >
using rocblas_real_t = typename rocblas_real_type< T >::type;

Then, you can write

rocblas_real_t< T > x_real = 78.0;

or 

using U = rocblas_real_t< T >.
U x_real = 78.0;

Reply:
I am not a big fan to specify typename each time you want to use the type.